source/plugin/workspace/idea.rst

```diff
@@ -5,11 +5,50 @@ Setting Up IntelliJ IDEA
 This article describes how to configure your **IntelliJ IDEA** workspace for plugin development with SpongeAPI and
 :doc:`a build system such as Maven or Gradle <../buildsystem/>` or the `Minecraft Development plugin <https://minecraftdev.org/>`_.
 
-Gradle
-======
+If you want to create your project completely from scratch, please skip ahead to the Gradle or Maven sections.  Using the Minecraft Dev plugin sets up a working starting point and eliminates some of the guesswork in getting your project off the ground.
 
-Creating Your Project
-~~~~~~~~~~~~~~~~~~~~~
+Using IDEA Minecraft Dev Plugin to Create a Working Starting Point
+==================================================================
+
+The `Minecraft Development plugin <https://minecraftdev.org/>`_ for IntelliJ is a great plugin by a community member which makes plugin project creation much easier while also providing some neat and useful features for development.  By default it will create a project which uses Gradle as the build tool.
+
+Installing the Minecraft Dev Plugin
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This plugin is available on the JetBrains IntelliJ plugin repository.
+
+Because of this, you can install the plugin through IntelliJ's internal plugin browser. Navigate to ``File -> Settings -> Plugins`` and click the ``Browse Repositories...`` button at the bottom of the window. In the search box, simply search for ``Minecraft``. You can install it from there and restart IntelliJ to activate the plugin.
+
+Creating Your Project from a Template
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Open **IntelliJ IDEA**.
+* Click ``Create New Project``.
+* Select ``Minecraft`` in the popup.
+* Make sure your **Project SDK** is set to some version of Java 8/1.8.
+* Select ``Sponge plugin`` for your project type, then click ``Next``.
+* Enter your **Group ID**, **Artifact ID**, and **Version**.
+
+  * Your **Group ID** should usually correspond to your Java package name. See :doc:`../plugin-class` for details.
+  * Your **Artifact ID** should usually correspond to your **plugin ID** you chose earlier, e.g. ``myplugin``.
+  * Your **Version** is up to you.
```

Comment:
Is this nitpick all that is holding up this PR? We can always clarify it later if we can get a better definition.

Reply:
If it matters, that stuff about group, artifact, and version are not my new text, just showing up as green in the diff because I moved paragraphs around, sorry for confusing matters that way.

So since that explanation of the version ID is already the status quo I don't see any reason to hold up this PR.  But I do agree that some additional guidance about appropriate version numbers would help (maybe could be added as a separate issue)

Reply:
Added #727